client/src/scss/partials/_header.scss

```diff
@@ -63,21 +89,22 @@ $sidebar-width: 300px;
     .header__logo-container {
         border-right: 1px solid darken($primary-color, 8%);
         height: 100%;
-	transition: background .25s;
+        transition: background .25s;
         overflow: hidden;
         width: $sidebar-width;
 
-	&:hover {
-	    background-color: darken($primary-color, 10%)
-	}
+        &:hover {
+        background-color: darken($primary-color, 10%)
+        }
     }
 
     .header__logo {
-	height: 100%;
-	background: url(/img/dashboard_logo.svg) 50% 50% no-repeat;
-	background-size: 180px;
+        background-size: 180px;
+        background: url(/img/dashboard_logo.svg) 50% 50% no-repeat;
+        height: 100%;
+        margin-left: rem-calc(43px);
         display: block;
-	margin-left: rem-calc(43px);
```

Comment:
@akrawchyk just went through and fixed indentation issues and also alphabetized class names